Formula/victoriametrics.rb

```diff
@@ -0,0 +1,76 @@
+class Victoriametrics < Formula
+  desc "Cost-effective and scalable monitoring solution and time series database"
+  homepage "https://victoriametrics.com/"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics/archive/v1.88.1.tar.gz"
+  sha256 "92803ae61f927b7173e25d4537c429bf3054edfa2f719bafc8fb9fcd411cab87"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV.deparallelize
+    mkdir_p buildpath/"src/github.com/VictoriaMetrics"
+    ln_sf buildpath, buildpath/"src/github.com/VictoriaMetrics/VictoriaMetrics"
+
+    system "make", "victoria-metrics"
+    bin.install "bin/victoria-metrics"
+
+    (bin/"victoriametrics_brew_services").write <<~EOS
+      #!/bin/bash
+      exec #{bin}/victoria-metrics $(<#{etc}/victoriametrics.args)
+    EOS
+
+    (buildpath/"victoriametrics.args").write <<~EOS
```

Comment:
If upstream doesn't ship configuration we shouldn't either

Reply:
I've used https://github.com/Homebrew/homebrew-core/blob/master/Formula/prometheus.rb#L42 as example and VictoriaMetrics use cmd flags for configuration without any config file.

Reply:
We shouldn't really do that with with prometheus either, but that config is much more limited in scope.

Reply:
Removed file creation and set args at service section.